source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -56,8 +41,12 @@ class ACRButton: FlatButton, ImageHoldingView {
             iconPositioned = iconPosition
         }
         initialize()
-        buttonStyle = style
-        setupButtonStyle(style: style)
+        buttonActionStyle = style
+        if config.isDarkMode {
+            buttonActionStyle = .darkMode
+        }
```

Comment:
This check is not required.....
Infact `ACRButton` can only have `ButtonsConfig` not `RenderConfig`